brianlong/www.validators.app
# 183466948 add active validators to data center stats

#### What's this PR do?
- Adds active validators count and active data centers count

#### How should this be manually tested?
- Run workers and confirm that they run without errors:
```
rails runner "DataCenterStatsWorker.perform_async('mainnet')"
rails runner "DataCenterStatsWorker.perform_async('testnet')"
```
- Compare validators_count and active_validators_count in generated stats

#### What are the relevant tickets?
- [URL to the PivotalTracker ticket](https://www.pivotaltracker.com/n/projects/2177859/stories/183466948)

#### Task completed checklist:
- [] Is there appropriate test coverage?
- [] Did I take into consideration possible security vulnerabilities?
- [] Are the controllers secure?
- [] Was every important matter documented?

#### Things to watch out for
- [N] Does this PR have any migrations?
- [N] Does this PR add new dependencies?
